client/webserver/api.go

```diff
@@ -2042,6 +2045,39 @@ func (s *WebServer) apiTakeAction(w http.ResponseWriter, r *http.Request) {
 	writeJSON(w, simpleAck())
 }
 
+// apiExportAppLogs time stamps the application log, zips it and sends it back to
+// the browser or webview as an attachment. Logfile names need to be distinct as
+// webview will not overwite an existing file.
+func (s *WebServer) apiExportAppLogs(w http.ResponseWriter, r *http.Request) {
+	timeStamp := time.Now().UnixMilli()
+	zipAttachment := fmt.Sprintf("attachment; filename=bwlog_%d.zip", timeStamp)
```

Comment:
A human readable timestamp would be nice for the file name, so you could know when you downloaded it.